extension-reference/mobile/analytics/internal-hits-for-adobe-analytics.md

```diff
@@ -25,29 +25,29 @@ Launch data includes information about the application's number of launches, day
 
 We also track the application version upgrades.
 
-### Crash Event
+### Crash Hit
 
 If your application is terminated without having first been backgrounded, the SDK reports a crash the next time your app is launched.
 
-This will be sent as an individual hit if the `backdateSessionInfo` flag is enabled in your configuration, otherwise it will be sent as part of the launch event.
+This will be sent as an individual hit if the `backdateSessionInfo` flag is enabled in your configuration, otherwise it will be sent as part of the launch hit.
 
 ### SessionInfo
 
 This hit contains information about the previous launch session, more specifically the session length.
 
-This will be sent as an individual hit if backdateSessionInfo flag is enabled in your configuration, otherwise it will be sent as part of the launch event.
+This will be sent as an individual hit if backdateSessionInfo flag is enabled in your configuration, otherwise it will be sent as part of the launch hit.
 
-## Identity Module
+## Identity Extension
 
-Here are the Analytics events that are triggered when the Identity module is enabled:
+Here are the Analytics hits that are triggered when the Identity extension is enabled:
 
 * Push
 
 The SDK tracks changes in the user's push notifications preference. A new hit is sent whenever the user opt-in/opt-out for push notifications.
 
-## Messages Module
+## Messages Extension
```

Comment:
Let's remove the messages extension from docs for now. We will add more details once campaign is released.